university-for-the-creative-arts-figures.csl

```diff
@@ -29,7 +29,16 @@
         <text term="figure" form="symbol" text-case="capitalize-first"/>
         <text variable="citation-number" suffix="."/>
         <text variable="title" font-style="italic"/>
-        <date date-parts="year" form="text" variable="issued"/>
+        <group>
+          <choose>
+            <if match="all" variable="issued">
+              <date date-parts="year" form="text" variable="issued"/>
+            </if>
+            <else>
+              <text value="s.d."/>
```

Comment:
@ibadger, we greatly prefer the use of CSL terms wherever possible. You can replace `<text value="s.d."/>` by `<text term="no date" form="short"/>`. You would also have to redefine the translation of this term in the locale section at the top of the style with `<term name="no date" form="short">s.d.</term>`.

(see https://github.com/citation-style-language/locales/blob/b01a5e4d420a4c0e446625c271144c52b0ab0a76/locales-en-GB.xml#L45 for the default translation of this term)